src/xdocs/config_whitespace.xml

```diff
@@ -1424,6 +1442,28 @@ import static java.math.BigInteger.ZERO;
   &lt;property name=&quot;tokens&quot; value=&quot;DOT&quot;/&gt;
   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+class Test {
+  int a, b;
+  Test(int a, int b) {
+    this.
+        a = a; // violation, linebreak after '.' is not allowed
```

Comment:
I really mean that java code has to be identical.
The difference should be only in trailing comments.

Reply:
done